modin/experimental/cloud/test/test_cloud.py

```diff
@@ -14,8 +14,111 @@
 import unittest.mock as mock
 import pytest
 from collections import namedtuple
+from inspect import signature
 from modin.experimental.cloud.rayscale import RayCluster
 from modin.experimental.cloud.cluster import Provider
+from ray.autoscaler.commands import (
+    create_or_update_cluster,
+    teardown_cluster,
+    get_head_node_ip,
+    _bootstrap_config,
+)
+
+
+@pytest.fixture
+def make_bootstrap_config_mock():
+    sig_bootstrap_config = signature(_bootstrap_config)
+
+    def bootstrap_config_mock(*args, **kwargs):
+        sig_bootstrap_config.bind(*args, **kwargs)
+        config = args[0]
+        config["auth"]["ssh_user"] = "modin"
+        config["auth"]["ssh_private_key"] = "X" * 20
+        return config
+
+    return bootstrap_config_mock
+
+
+@pytest.fixture
+def make_get_head_node_ip_mock():
+    sig_get_head_node_ip = signature(get_head_node_ip)
+
+    def get_head_node_ip_mock(*args, **kwargs):
+        sig_get_head_node_ip.bind(*args, **kwargs)
+        return "127.0.0.1"
+
+    return get_head_node_ip_mock
+
+
+@pytest.fixture
+def make_teardown_cluster_mock():
+    sig_teardown_cluster = signature(teardown_cluster)
+
+    def teardown_cluster_mock(*args, **kwargs):
+        sig_teardown_cluster.bind(*args, **kwargs)
+
+    return teardown_cluster_mock
+
+
+@pytest.fixture
+def make_create_or_update_cluster_mock():
+    sig_create_or_update_cluster = signature(create_or_update_cluster)
+
+    def create_or_update_cluster_mock(*args, **kwargs):
+        sig_create_or_update_cluster.bind(*args, **kwargs)
+
+    return create_or_update_cluster_mock
+
+
+def test__bootstrap_config(make_bootstrap_config_mock):
+    with mock.patch(
+        "modin.experimental.cloud.rayscale._bootstrap_config",
+        make_bootstrap_config_mock,
+    ):
+        RayCluster(Provider(name="aws"))
+
+
+def test_get_head_node_ip(make_bootstrap_config_mock, make_get_head_node_ip_mock):
+    with mock.patch(
+        "modin.experimental.cloud.rayscale._bootstrap_config",
+        make_bootstrap_config_mock,
+    ):
+        ray_cluster = RayCluster(Provider(name="aws"))
```

Comment:
this looks like it should be a fixture (using another fixture), as it's repeated quite a while in tests